api/v1alpha3/azurecluster_types.go

```diff
@@ -50,6 +50,8 @@ type AzureClusterSpec struct {
 type AzureClusterStatus struct {
 	Network Network `json:"network,omitempty"`
 
+	FailureDomains clusterv1.FailureDomains `json:"failureDomains,omitempty"`
```

Comment:
let's add a comment here to describe how the field is used

Reply:
Also, can you please add a short doc in docs/topics to explain to use Failure Domains with Azure?

Reply:
@richardcase see the two comments above about docs ^

Reply:
Ah sorry @CecileRobertMichon, i missed these comments. I will make the changes.

Reply:
Docs added.